client/webserver/site/src/js/markets.ts

```diff
@@ -1742,7 +1742,7 @@ export default class MarketsPage extends BasePage {
   handleBookRoute (note: BookUpdate) {
     app().log('book', 'handleBookRoute:', note)
     const mktBook = note.payload
-    const [b, q] = [app().assets[this.market.base.id], app().assets[this.market.quote.id]]
```

Comment:
This was the culprit and reading initial from app() which we’ve been doing elsewhere. Thanks for fixing. AFK so can’t run it now but looks gtg.